data/campaigns/The_Rise_Of_Wesnoth/scenarios/07_Return_to_Oldwood.cfg

```diff
@@ -117,7 +117,7 @@
         [/message]
         [message]
             speaker=Prince Haldric
-            message= _ "Yes, it was a tough battle, but we prevailed. Now, I’m having a problem. I know not what I should do next."
+            message= _ "Yes, it was a tough battle, but we prevailed. Now, I have a problem. I know not what I should do next."
```

Comment:
> I know not what I should do next.

Wouldn't "I do not know what I should do next" be better?

Reply:
Not necessarily.
Have you heard the phrase "You know not what you are asking?"

Reply:
> Wouldn't "I do not know what I should do next" be better?

They are both correct grammar and mean the same thing, but the existing phrasing is archaic.